### PR TITLE
add GSoc 2024 info

### PR DESCRIPTION
**Pull request by hegner**

### CODE REVIEW

File: _activities/gsoc.md [hunk 1]
```diff
@@ -24,22 +24,22 @@ To answer these questions, particle physicists build software to simulate and an
 
 The CERN software for experiments (CERN EP-SFT) group has participated in the GSoC since 2011. Since 2017 the program has expanded to involve the high-energy physics community under the umbrella of the HEP Software Foundation.
 
-Information from last year’s GSoC can be found [here]({{ site.baseurl }}/gsoc/2022/index.html).
+Information from last year’s GSoC can be found [here]({{ site.baseurl }}/gsoc/2023/index.html).
```
Comment (graeme-a-stewart) [inline _activities/gsoc.md:27]: Not Found

`/gsoc/2023/index.html' not found.

File: _activities/gsoc.md [hunk 2]
```diff
@@ -49,100 +49,100 @@ For further information, feel free to contact the HSF GSoC admins or join our op
 
 ### Student pages
 
-The CERN-HSF 2022 student blogs are now available. [Check them out!]({{ site.baseurl }}/gsoc/2022/blogs.html)
+The CERN-HSF 2023 student blogs are now available. [Check them out!]({{ site.baseurl }}/gsoc/2023/blogs.html)
```
Comment (graeme-a-stewart) [inline _activities/gsoc.md:52]: There is only one blog post from 2023, so maybe we need to remove this link? I am not sure it gives a great impression.

File: _activities/gsoc.md [hunk 3]
```diff
@@ -49,100 +49,100 @@ For further information, feel free to contact the HSF GSoC admins or join our op
...
 
-Student blogs are mandatory for receiving a positive evaluation in CERN-HSF starting with 2022 GSoC edition! If you are a CERN-HSF student, please read this [blog]({{ site.baseurl }}/gsoc/blogs/2022/blog_HSF_TheAdmins.html).
+Student blogs are mandatory for receiving a positive evaluation in CERN-HSF! If you are a CERN-HSF student, please read this [blog]({{ site.baseurl }}/gsoc/blogs/2022/blog_HSF_TheAdmins.html).
```
Comment (graeme-a-stewart) [inline _activities/gsoc.md:54]: Apparently this is not true!

File: gsoc/students-guideline.md [hunk 4]
```diff
@@ -12,13 +12,13 @@ Get started checking out the [Google Summer of Code Guides for students](https:/
...
   <!-- Next <br><br>, add an extra new line, otherwise there is no space between point 2 and previous nested bullet  -->
   <br><br>
 
-2. Once you have checked all our proposals and picked the one you like most, please contact the corresponding mentors (their emails can be found at the end of each proposal) sending them your CV and the motivation for picking their proposal. Please do this only after the list of accepted Organizations is published on `March 7`, but not after `March 20`, to give time to mentors to evaluate your technical skills. You are encouraged to apply to a maximum of two project proposals.
+2. Once you have checked all our proposals and picked the one you like most, please contact the corresponding mentors (their emails can be found at the end of each proposal) sending them your CV and the motivation for picking their proposal. Please do this only after the list of accepted Organizations is published on `Feb 21`, but not after `March 17`, to give time to mentors to evaluate your technical skills. You are encouraged to apply to a maximum of two project proposals.
```
Comment (graeme-a-stewart) [inline gsoc/students-guideline.md:21]: Minor comment: Font choice is rather odd. How about **bold** or *italic*? This is generally true of this page.

**Review by graeme-a-stewart — CHANGES_REQUESTED** [review]
Thanks @hegner ! A few comments.

File: _activities/gsoc.md (hunk 1 above)
Comment (hegner) [reply _activities/gsoc.md:27]: fixed. thanks for spotting

File: _activities/gsoc.md (hunk 3 above)
Comment (hegner) [reply _activities/gsoc.md:54]: indeed. we haven't been pushy enough last time

**Review by graeme-a-stewart — APPROVED** [review]

**Pull request merged**